examples/wannier90.jl

```diff
@@ -11,21 +11,28 @@
 #     Bugs are likely and we welcome issues in case you find any!
 #
 # This example shows how to obtain the MLWFs corresponding
-# to the first eight bands of silicon. Since the bands 5 to 8 are entangled,
-# 12 bands are first computed to obtain 8 MLWFs by a disantanglement procedure.
+# to the first five bands of graphene. Since the bands 2 to 11 are entangled,
+# 15 bands are first computed to obtain 4 MLWFs by a disantanglement procedure.
```

Comment:
Do you mean 5 here?

Reply:
Hm, you have 4 below. @LaurentVidal95 I thought 5 was the correct number here?

Reply:
Ah yes that's correct I think I forgot to account for a degeneracy in the bands.